server/src/main/java/org/tctalent/server/api/admin/CandidateExamAdminApi.java

```diff
@@ -57,6 +62,38 @@ public class CandidateExamAdminApi implements IJoinedTableApi<CreateCandidateExa
         return candidateExamDto().build(candidateExam);
     }
 
+    /**
+     * Get all records joined to the given parent
+     * @param parentId ID of parent record
+     * @return All records associated with the given parent
+     */
```

Comment:
This is duplicate documentation from the interface. If the interface is documented you shouldn't just duplicate it.
Normally when documenting these overriden interface methods, we change the generic documentation to match the particular class. So parentId becomes candidateId and text referring to "records" becomes CandidateExams, parent becomes Candidate.

That will be much more useful to other programmers.

Reply:
Done